exercises/toRoman/toRoman3.js

```diff
@@ -0,0 +1,38 @@
+function toRoman3(num) {
+  let romanDict = [
+    [1000,'M'],
+    [900,'CM'],
+    [500,'D'],
+    [400,'CD'],
+    [100,'C'],
+    [90,'XC'],
+    [50,'L'],
+    [40,'XL'],
+    [10,'X'],
+    [9,'IX'],
+    [5,'V'],
+    [4,'IV'],
+    [1,'I'],
+  ];
+  let workingNum = num;
+  let inRoman = ''
+  for (let val of romanDict) {
+    for (let i = 0; i < Math.floor(workingNum/val[0]); i++) {
```

Comment:
Consider:

```javascript
function divmod(n, d) {
  return [Math.floor(n / d), n % d];
}

for (let [val, roman] of romanDict) {
  [workingNum, inRoman] = [workingNum % val, roman.repeat(workingNum / val)];
}
```

As a fold (JS does not make this one look nice):

```javascript
romanDict.reduce(([workingNum, inRoman], [val, roman]) => [workingNum % val, inRoman + roman.repeat(num / val)]), [num, ''])[1]
```

Typing this in GitHub, I might've made a mistake. I tested it outside of GH and they both work as expected.